HS - Add an Endpoint for an Individual Review

hannuhsu opened the pull request:
Added a new GET /api/reviews/get?id={id} endpoint in ReviewController that retrieves a single review by its ID and enforces that only the review’s author or an admin may access it. Wrote unit tests in ReviewControllerTests.java to validate owner access, admin override, forbidden access for other users, and proper 404 handling for missing reviews.

Closes #11

tiffanyxgu commented:
There's a line of commented out code at the top of menuItemFixtures.js that you should probably delete if it's not being used for anything. Otherwise, the code looks fine to me.

Can you add a link to your personal dokku deployment so it we can see that it works on there too?

hannuhsu commented:
package lock: pretty sure it's because i updated the package last commit, fixed it, but ran prettier on my files so i'm guessing i added an extra empty line and prettier just fixed it? 

will update dokku once i get support - updates not working for some reason
